bundle-hack/update_csv.go

```diff
@@ -224,14 +226,43 @@ func replaceImages(csv map[string]interface{}) {
 	fmt.Println("Updated the deployment manifest to use downstream builds")
 }
 
-func removeRelated(csv map[string]interface{}) {
+func updateRelatedImages(csv map[string]interface{}) {
 	spec, ok := csv["spec"].(map[string]interface{})
 	if !ok {
 		log.Fatal("Error: 'spec' does not exist in the CSV content")
 	}
 
-	delete(spec, "relatedImages")
-	fmt.Println("Removed the operator from operator manifest")
+	// Extract SHAs from Konflux pull specs and build Red Hat registry URLs
+	imageShaOperator := getPullSpecSha(konfluxOperatorPullSpec)
+	imageShaOpenscap := getPullSpecSha(konfluxOpenscapPullSpec)
+	imageShaContent := getPullSpecSha(konfluxContentPullSpec)
+	imageShaMustGather := getPullSpecSha(konfluxMustGatherPullSpec)
+
+	delimiter := "@"
+	registryPrefix := "registry.redhat.io/compliance/"
+
+	// Create or update the relatedImages section with Red Hat registry images
+	relatedImages := []map[string]string{
+		{
+			"name":  "openscap",
+			"image": registryPrefix + "openshift-compliance-openscap-rhel8" + delimiter + imageShaOpenscap,
+		},
+		{
+			"name":  "operator",
```

Comment:
In older CVS files, we also had `compliance-operator` as a related image, even though it was duplicated with this image.

Are we just going to build without it and see if we need it?

Reply:
I guess that depends on how we refer to our related images in CO code:
https://docs.redhat.com/en/documentation/openshift_container_platform/4.15/html/operators/developing-operators#olm-enabling-operator-for-restricted-network_osdk-generating-csvs